### PR TITLE
Fix #3: Allows quick-sip to be run multiple times in one build process

### LINKED ISSUE #3: Allow quick-sip to run multiple times in the same build
There are at least two implementation details that prevent this from happening:

1. "options.js" returns a global options object
2. "createBundleTasks.js" sets a global flag "initialized"

### PR DESCRIPTION
**Pull request by Earthstar**
Options generates a new options object each time
createBundleTasks initialized once per task

### CODE REVIEW

File: tasks/build-app.js [hunk 1]
```diff
@@ -20,8 +17,8 @@ module.exports = function(gulp, newBrowserifyOptions) {
       } else {
         browserifyBundler.transform(transform);
       }
-    })
-  };
+    });
+  }
```
Comment (mandragorn) [inline tasks/build-app.js]: this should have a ; too since it is an = statement.

File: tasks/build-app.js [hunk 2]
```diff
@@ -41,7 +38,7 @@ module.exports = function(gulp, newBrowserifyOptions) {
       .pipe($.util.env.type === 'production' ? $.uglify() : $.util.noop())
       .pipe($.util.env.type !== 'production' ? $.sourcemaps.write('./') : $.util.noop())
       .pipe(gulp.dest(options.browserify.dist));
-  };
+  }
```
Comment (mandragorn) [inline tasks/build-app.js]: revisit the removal of ; throughout this commit.  I'm not sure they all should be removed, but it is hard to see in this format so I may be wrong.

**Comment by mandragorn** [comment]
did you verify that the documentation is still correct after this update?  (if not update the readme).

File: tasks/build-app.js (hunk 2 above)
Comment (Earthstar) [inline tasks/build-app.js]: my jslint says it's redundant to put semicolons after function declarations

Comment (mandragorn) [inline tasks/build-app.js]: ok, nvm, then.

**Pull request merged**